actionpack/lib/action_dispatch/system_test_case.rb

```diff
@@ -1,5 +1,7 @@
 # frozen_string_literal: true
 
+gem "capybara", "~> 2.13"
```

Comment:
This has a side effect of users not being able to update to Capybara 3 when it releases, since the requirement is in non user modifiable source.  Wouldn't it be better as `gem "capybara", ">= 2.13"` in this file?

Reply:
I submitted it this way as that's what was requested on #30952.  I'm assuming the preferred convention here is to match rubygem's preference that open-ended dependencies are to be avoided.

Reply:
It is fine to not allow update to capybara 3 since rails may not work with that version, but yeah, it should be 2.15. I'll update it.

Reply:
@rafael It will work with 3 since 3 is just going to be a remove deprecations and drop ruby versions release.  However, this is adding a requirement just for a warning in a non user editable file - The more restrictive requirement in the Gemfile generated for an app is correct, and where the actual restriction should be (and users can modify that when Capybara 3 is released).  I really feel this should be `>= 2.15` in this file.

Reply:
@twalpole so I should be good to use `gem 'capybara', '~> 3.0.2', '< 4.0'` in my Rails 5.1 app? I'm experiencing https://github.com/rails/rails/issues/30952

Reply:
@MrHubble If you're using Rails 5.1 you should be able to use `gem 'capybara', '~> 2.18'` in your gem file since Rails 5.1 won't allow Capybara 3+ due to  this change (if you're using system tests).  With Rails 5.2 you should be good to use `gem 'capybara', `~> 3.0`` (note that specifying the `<4.0` is unnecessary due to the the way `~>` works

Reply:
Great. Thanks so much for taking the time to clarify.